code/__HELPERS/global_lists.dm

```diff
@@ -36,6 +36,11 @@
 		var/datum/superheroes/S = new T
 		all_superheroes[S.name] = S
 
+	paths = subtypesof(/datum/nations)
+	for(var/T in paths)

```

Comment:
not much you can actually do about this but all of these should really be handled by some type of generic "all subtypes of X go into Y list"